charts/hcloud-fip-controller/values.yaml

```diff
@@ -67,6 +67,7 @@ existingConfigMap: hcloud-fip-controller-config
 # configuration ConfigMap as part of the release, and
 # existingConfigMap is ignored.
 #
+configInline: {}
 # Refer to https://github.com/cbeneke/hcloud-fip-controller/ for
 # available options.
-configInline: {}
+hcloudAPIToken: ""
```

Comment:
This change would force the user to add the (secret) api token to the values.yaml. I would prefer to add a toggle (e.g.`existingSecretName`) which -if set- is used instead of creating the secret via helm. I'm fine with it being empty per default though.